src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx

```diff
@@ -129,4 +129,7 @@
   <data name="WorkloadListFooter" xml:space="preserve">
     <value>Use `dotnet workload search` to find additional workloads to install.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="WorkloadListHeader" xml:space="preserve">
+    <value>This command lists only workloads that were installed via `dotnet workload install` in this version of the SDK and not those that were installed via Visual Studio.</value>
```

Comment:
```suggestion
    <value>This command lists only workloads that were installed via `dotnet workload install` in this version of the SDK and not those that were installed via Visual Studio Installer.</value>
```
?

Reply:
I'm going to leave this as is for now as Kathleen/PMs came up with the wording and have already approved offline.

Reply:
<value>This command lists only workloads that were installed via `dotnet workload install` for this version of the SDK. It does not list workloads that were installed via Visual Studio Installer.</value>

I found the long sentence hard to read, and think Nathan's addition of the word installer is good.